public/list.css

```diff
@@ -0,0 +1,43 @@
+@import url('https://fonts.googleapis.com/css?family=Fascinate+Inline|Luckiest+Guy|Great+Vibes');
```

Comment:
When loading fonts, remember that sometimes they are too big. For example if you need only `400` weight, select only this one. If app is in English, include only 'latin' pack. Please check this